### PR TITLE
Account choice

### PR DESCRIPTION
**Pull request by lcarcone**
Hi Tripu,

I have added the page 'Use an account or not' and I have modified it's style.

Thanks,
Laurent

### CODE REVIEW

File: css/site-wbs-style.css [hunk 1]
```diff
@@ -629,6 +629,37 @@ span.required {
...
+  list-style: none;
+}
+
+ul.account li.main {
+  background: #eee;
+  color: #000;
+  margin: 10px 10px 15px 0px;
+}
+
+ul.account li.main:hover {
```
Comment (tripu) [inline css/site-wbs-style.css]: I'd remove the hover effect; I think it's more confusing than anything.
(It's useful on long tables and things like that to help the user know what row they're pointing at; but this is a small selection among two options only.)

File: html/account-choice.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+   <div id="Content">
+
+<form class='login' action=''>
+<ul class='account'>
+<li><strong>Do you have a W3C Web account?</strong>
+<ul>
+<li class='main'>
+<p><strong>
+<a href='https://www.w3.org/2002/09/wbs/99999/wbs-types-test/?login'>YES</a> &#xA0; </strong>
```
Comment (tripu) [inline html/account-choice.html]: I'd remove the hyperlink in &ldquo;YES&rdquo;; it's redundant and a bit confusing. Also, there is no hyperlink in &ldquo;NO&rdquo;.

**Review by tripu — CHANGES_REQUESTED** [review]
:+1: except for those two tiny suggestions.

**Comment by tripu** [comment]
@lcarcone, what do you think about my comments above?

**Comment by lcarcone** [comment]
Hi Tripu, ok with both suggestions

**Comment by lcarcone** [comment]
Hi tripu,
I took your suggestions into account.
Thanks

**Review by tripu — APPROVED** [review]

**Pull request merged**